gatsby-node.js

```diff
@@ -115,7 +115,9 @@ async function parseSVG({
       source.contentful_id + ': ' + absolutePath
     )
   }
-  const { data: optimizedSVG } = await svgo.optimize(svg)
+  const { data: optimizedSVG } = await svgo.optimize(svg, {
+    path: absolutePath
+  })
```

Comment:
Awesome.Was first wondering where the magic happens, but you pointet it out very well in your PR description